x-pack/legacy/plugins/apm/server/lib/transactions/distribution/get_buckets/transform.ts

```diff
@@ -39,7 +39,7 @@ export function bucketTransformer(response: DistributionBucketResponse) {
   ).map(getBucket);
 
   return {
-    totalHits: response.hits.total,
+    totalHits: response.hits.total.value,
```

Comment:
I think this is only used to determine whether there are any hits or not:
```js
const noHits = totalHits === 0
```

In which case this is fine. But just make sure we don't need accurate counts. Same with the other places.

Reply:
Actually, can you change all the endpoints that return `totalHits` to return a boolean `noHits` instead so it's clear that they don't need accurate counts?